resources/assets/less/bem/search-result-more.less

```diff
@@ -28,7 +28,7 @@
   color: @osu-colour-c1;
   font-size: @font-size--title-small-3;
 
-  &:hover {
+  &--active {
     background-color: @osu-colour-b2;
     color: @osu-colour-c1;
```

Comment:
`color` still needs to be set on hover, otherwise the text will change colour on hover